syntax = "proto3";

package predictionmarkets.marketplace;

message MarketInfo {
    string name = 1;
    string details = 2;
}

message ListMarketsRequest {
    int32 limit = 1;
    int64 offset = 2;
}

message ListMarketsResponse {
    map<string, MarketInfo> market_infos = 1;
}
